chore: Added Gravestone game objects

DanielEmmanuel1 opened the pull request:
Summary
- Duplicated gravestone gameobject
- Repositioned all duplicates accordingly

DanielEmmanuel1 commented:
@jimenezz22 All done
Please review

jimenezz22 reviewed (commented):
Can you provide evidence of the changes you made and a detailed summary of what you did? I see you are not following our pull request template

DanielEmmanuel1 commented:
Changes i made
- I imported the reference image into unity
- Then I drop it as the first child of the Grid gameobject making sure it perfectly fits the grid
- Then I disabled the Ground object so I can see the reference image in the background
- Then I duplicated the Grave objects and positioned them over the graves in the reference image
- Then I deleted the reference image and set the re-enabled the Ground object
- Then I deleted the reference image from the project

@jimenezz22 I didn't read the readme indepthly, that was an oversight from my part and i do apologise for that, I just went through the readme and if need be, i can still create a branch, push from there and create a new PR, whatever works for u.

jimenezz22 commented on Assets/Scenes/DarkForest.unity:
This looks like you removed the scene from the pr

DanielEmmanuel1 replied:
I'll handle this now

DanielEmmanuel1 commented:
Hi @jimenezz22 

I think the issue here is someone else made updates to the scene while I was making changes to it too
And now there's a conflict because as u know GitHub cannot merge Unity scenes.

A feasible solution will be to discard/revert my changes, then I'll pull and implement the changes again
But you have to let me know as soon as you have reverted the changes to avoid another scene conflict

Is this okay?

DanielEmmanuel1 commented:
@jimenezz22 Please can u check now

jimenezz22 reviewed (approved):
pr already tested on my local repo and works successfully
